oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -60,18 +60,15 @@ impl LogNode {
             // An error indicates the Runtime is terminating. We ignore it here and keep trying to
             // read in case a Wasm Node wants to emit remaining messages. We will return
             // once the channel is closed.
-
             let _ = self.runtime.wait_on_channels(&[Some(self.reader)]);
 
             if let Some(message) = self.runtime.channel_read(self.reader)? {
                 match LogMessage::decode(&*message.data) {
-                    Ok(msg) => info!(
-                        "{} LOG: {} {}:{}: {}",
-                        pretty_name,
-                        to_level_name(msg.level),
-                        msg.file,
-                        msg.line,
-                        msg.message
+                    Ok(msg) => log!(
+                        target: &format!("{}:{}", msg.file, msg.line),
+                        to_level(msg.level),
+                        "{}",
+                        msg.message,
```

Comment:
This makes all log lines from runtime and from the application look very similar. Do you think we should try to separate runtime and application logs at some point, perhaps redirecting output to different files?
I guess a less drastic solution would be to use more structured logging, and tag each log entry with whether it came from the runtime or the application; that way at least it may be easy to filter them out in a log viewer.
@rbehjati do you have any opinion on this? Is there a way in kubernetes to have multiple output logs?

Reply:
GCP/kubernetes can handle logs from different resources, and the API allows writing logs to different files too. But I don't know how you could write to different files with the `log` macro. 
If that is not possible, we could tag the log entires as you suggested. I guess it is sufficient to only tag the one here.